.golangci.yaml

```diff
@@ -0,0 +1,130 @@
+run:
```

Comment:
I've been made aware that there's now a v2 config format. I'm not sure if you want to convert this to that format now, some point in the future, or not at all

Reply:
yeah, saw that myself.  It'll have to wait for a future point in time.